_attributes/common-attributes.adoc

```diff
@@ -95,6 +95,10 @@ endif::[]
 :odo-title: odo
 //alibaba cloud
 :alibaba: Alibaba Cloud
+//OpenShift Kubernetes Engine
+:oke: OpenShift Kubernetes Engine
+//OpenShift Platform Plus
+:opp: OpenShift Platform Plus
 //openshift virtualization (cnv)
```

Comment:
@kalexand-rh you OK with these values? I also was not sure of how this file was organized, so if you believe the attributes should be defined elsewhere, please just let me know. Thanks in advance.

Reply:
LGTM!